database/src/main/postgres/runs/create_partitioning_if_not_exists.sql

```diff
@@ -50,11 +52,31 @@ DECLARE
     _status                 BIGINT;
 BEGIN
 
+    SELECT partit_validity.status
```

Comment:
I would put no validation to this function, and actually any function. Let's put a check (`_is_partitioning_valid`/`is_partitioning_valid`) into the table `runs.partitionings` on field `partitioning`. That will ensure consistency of data always.

Reply:
yep, will change it